cmd/meroxa/turbine/utils.go

```diff
@@ -214,6 +221,11 @@ func RunCmdWithErrorDetection(_ context.Context, cmd *exec.Cmd, _ log.Logger) (s
 			if stdOutMsg != "" {
 				errLog = stdOutMsg + errLog
 			}
+
+			if strings.Contains(errLog, "rpc error") {
+				errLog = clarifyGrpcErrors(errLog)
```

Comment:
any chance stdout would be helpful here?

Reply:
the entirety of stdout for this error is the grpc error. So we could:
- include the grpc error along with this clarified, actionable error (in case that stdout is useful for a different, concurrent issue)
- trim the grpc error from stdout, and add it to the actionable error
- replace stdout entirely with this clarified error (current approach)

Reply:
ok, I landed on a middle-ground solution that gives us some flexibility. I added a `log.Debug()` above when we are doing this replacing. that way, if we are debugging, we still have visibility into the full stdout, without adding more error-specific logic.

with `--debug`:
```
~/go/src/notion-s3 main
❯ ../cli/meroxa apps deploy --debug
Validating your app.json...
        ✔ Checked your language is "golang"
        ✔ Checked your application name is "notion-s3"
Checking for uncommitted changes...
        ✔ No uncommitted changes!
<redacted prior steps for conciseness>
2023/08/08 15:19:27 rpc error: code = Unknown desc = invalid ProcessCollectionRequest.Collection: embedded message failed validation | caused by: invalid Collection.Name: value length must be at least 1 runes
exit status 1
Error: missing source or source collection, please ensure that you have configured your source correctly:
https://docs.meroxa.com/turbine/troubleshooting#troubleshooting-checklist"
```

without `--debug`:
```
~/go/src/notion-s3 main
❯ ../cli/meroxa apps deploy        
Validating your app.json...
        ✔ Checked your language is "golang"
        ✔ Checked your application name is "notion-s3"
Checking for uncommitted changes...
        ✔ No uncommitted changes!
Error: missing source or source collection, please ensure that you have configured your source correctly:
https://docs.meroxa.com/turbine/troubleshooting#troubleshooting-checklist"
```